javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,56 @@
+const width = 4; // the integer '4' is being assigned to 'width'
+const height = 7; // the integer '7' is being assigned to 'height'
+const area = width * height; // the expression 'width * height' value is being assigned to 'area'
+
+console.log('value of area:', area); // the log method of the console object is being called with 2 arguments a string and 'area'
+console.log('typeof area:', typeof area); // the log method of the console object is being called with 2 arguments a string and typeof 'area'
+
+const bill = 10; // the integer '10' is being assigned to 'height'
+const payment = 4.5; // the float '4.5' is being assigned to 'payment'
+
+const change = bill - payment; // the expression 'bill - payment' value is being assigned to 'area'
```

Comment:
Your `payment` should be larger than the `bill`, otherwise you didn't pay the full tab 😂

Reply:
Ops 😂 Thank you!